Packs/CommonScripts/Scripts/EnrichExcludeButton/EnrichExcludeButton.yml

```diff
@@ -16,3 +16,4 @@ engineinfo: {}
 tests:
 - No tests (auto formatted)
 fromversion: 8.8.0
+comment: This script is only meant to be used for the inaction of the button in an indicator that is Enrich Excluded, it should not be used otherwise.
```

Comment:
```suggestion
comment: This script is only meant to be used to disable the Enrich Excluded button in an indicator. It should not be used otherwise.
```